CHANGELOG.md

```diff
@@ -40,6 +40,10 @@ Semantic versioning in our case means:
 - Adds baseline information for all complexity violation messages: `x > baseline`
 - Changes how cognitive complexity is calculated
 - Adds support for positional arguments in different checks
+- Forbids to use `continue` and `break` in `finally`. It is a terrible practice, because
```

Comment:
This change is unrelated. Please, make sure you didn't commit extra files (there are more than 300 files changed!)